tests/test_dbt.py

```diff
@@ -45,6 +45,10 @@ def test_integration_basic_dbt(self):
             # 1 with a diff
             assert diff_string.count("  Rows Added    Rows Removed") == 1
 
+    @unittest.skipIf(
+        os.environ.get("MOTHERDUCK_TOKEN") is None,
```

Comment:
This behavior is consistent with forked branch contributions as we simply omit which databases are tested based on whether database secrets are available or not. 

Example PR: https://github.com/datafold/data-diff/actions/runs/7264201391/job/19791203217?pr=806#step:8:65

You'll notice databases like snowflake and presto are omitted from the tested databases because it's a forked branch PR.

Vs. this native branch PR: https://github.com/datafold/data-diff/actions/runs/7350253086/job/20011581079